lib/runtime-c-api/Cargo.toml

```diff
@@ -24,12 +24,24 @@ default-features = false
 path = "../runtime-core"
 version = "0.6.0"
 
+[dependencies.wasmer-middleware-common]
+path = "../middleware-common"
+version = "0.6.0"
+optional = true
+
+[dependencies.wasmer-singlepass-backend]
+path = "../singlepass-backend"
+version = "0.6.0"
+optional = true
+
+
 [features]
 default = ["cranelift-backend"]
 debug = ["wasmer-runtime/debug"]
 cranelift-backend = ["wasmer-runtime/cranelift", "wasmer-runtime/default-backend-cranelift"]
 llvm-backend = ["wasmer-runtime/llvm", "wasmer-runtime/default-backend-llvm"]
 singlepass-backend = ["wasmer-runtime/singlepass", "wasmer-runtime/default-backend-singlepass"]
+metering = ["wasmer-runtime/singlepass", "wasmer-middleware-common", "wasmer-singlepass-backend"]
```

Comment:
Since `singlepass-backend` is already a feature, you can simplify the “feature dependency” by writing:

```toml
metering = ["singlepass-backend", "wasmer-middleware-common"]
```

Reply:
I did that. It then sets `wasmer-runtime/default-backend-singlepass`.
and the default features set `wasmer-runtime/default-backend-cranelift`.

Setting two default backends caused a panic or compile issue. (I can look up the line if you want). I had to set this to avoid crashes. Ideal would be to disable the cranelift-backend completely for the metered build I guess. But I don't know how to disable default features.

Reply:
Also note that singlepass-backend wouldn't even compile without an extra dependency added. Or maybe that is the problem. Hmmm...